amundsen_rds/models/user.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from sqlalchemy import BigInteger, Boolean, Column, Integer, String
+from sqlalchemy.orm import relationship
+
+from amundsen_rds.models.base import KEY_LEN, PUBLISHED_TAG_LEN, Base
+
+
+class User(Base):
+    """
+    User model.
+    """
+    __tablename__ = 'users'
+
+    rk = Column(String(KEY_LEN), primary_key=True)
+    email = Column(String(320), nullable=False)
+    is_active = Column(Boolean)
```

Comment:
is nullable default to true?

Reply:
Without the nullable argument, the column can allow null value